portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,53 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Haihan's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="firstpage">
+        <div class="app-header">
+            <div class="app-header-right">
+                <ul><a href="#description">About Me</a></ul>
+                <ul><a href="https://github.com/konzert?tab=repositories" target="_blank">Projects</a></ul>
```

Comment:
Generally the links in header should lead to different sections on the page. This one broke my expectation by jumping to github.

Reply:
Added a project section on page